docs/_plugins/search_index.rb

```diff
@@ -389,6 +388,16 @@ def is_latest?(group, model)
   force_reindex = editions_changed?(editions)
   bulk_indexer = BulkIndexer.new(client)
 
+  # remove renamed or deleted posts from the index
```

Comment:
Why do we need to move this block of code?

Reply:
Moving this block of code i.e removing the changed posts from elastic search by their urls will remove the old cards that have their edition changed.